docs/guide/development-setup.md

```diff
@@ -12,60 +12,91 @@ videoList:
+**For macOS, Linux, and WSL:**
+Use NVM (Node Version Manager). You can find the installation instructions [here](https://github.com/nvm-sh/nvm?tab=readme-ov-file#installing-and-updating).
 
-⚠️ NVM is only available for Linux / Mac OS / Windows WSL.
+**For Windows (native):**
+Use NVM for Windows. Installation instructions are available [here](https://github.com/coreybutler/nvm-windows#installation--upgrades).
 
-Once installed, you can switch between different versions of Node and npm by using
+Once installed for any operating system, you can switch between different versions of Node and npm:
 
 ```bash
-$ nvm install 18
+$ nvm install 22
 # and
-$ nvm use 18
+$ nvm use 22
 ```
 
-See the `nvm` page for more options.
-
-### 2. MongoDB 6.0+<br>
+### 2. MongoDB 6.0+
 
 You can make a MongoDB instance available to your project in three ways:
 
+**Option 1: MongoDB Atlas**
+
 MongoDB offers a hosted version of the server, [MongoDB Atlas](https://www.mongodb.com/atlas/database), that offers a free tier and doesn't require any local software installation. You can set a connection string for a hosted instance using the `APOS_MONGODB_URI` environment variable or by setting the options of the [`@apostrophecms/db` module](/reference/modules/db.html) at the project level.
 
 For example:
 ```bash
 export APOS_MONGODB_URI="mongodb+srv://username:pa%24%24word@mycluster.1234x.mongodb.net/YOUR-PROJECT-NAME?retryWrites=true&w=majority"
 ```
 
-For offline local development, is to use Docker to host the server. You can follow our instructions [here](/guide/dockerized-mongodb.md) and then skip to the next [section](/guide/development-setup.md#installing-the-apostrophe-cli). By default, Apostrophe attempts to connect to the database using the connection string `mongodb://localhost:27017/<project-shortName>` where the `shortName` is set in the project `app.js` file. The Docker tutorial sets the MongoDB container up to use this port, so no changes are needed.
+**Option 2: Docker**
+
+For offline local development, you can use Docker to host the server. You can follow our instructions [here](/guide/dockerized-mongodb.md) and then skip to the next [section](/guide/development-setup.md#installing-the-apostrophe-cli). By default, Apostrophe attempts to connect to the database using the connection string `mongodb://localhost:27017/<project-shortName>` where the `shortName` is set in the project `app.js` file. The Docker tutorial sets the MongoDB container up to use this port, so no changes are needed.
 
-The final option, also for local development, is to install the MongoDB community edition server. As with the Docker container, the community edition server uses port 27017 and Apostrophe will connect to the MongoDB instance without any additional changes.
+**Option 3: MongoDB Community Edition**
```

Comment:
For Windows users not using WSL, Community Edition is arguably as easy as Atlas. Click boom... I think that should be mentioned.